libcnb-test/src/test_context.rs

```diff
@@ -199,6 +244,20 @@ impl<'a> TestContext<'a> {
     }
 }
 
+/// Downloaded SBOM files.
+pub struct SbomFiles {
+    sbom_files_directory: PathBuf,
+}
+
+/// The type of SBOM.
+///
+/// Not to be confused with [`libcnb_data::sbom::SbomFormat`].
+pub enum SbomType {
+    Launch,
```

Comment:
Should the `Launch` variant also have rustdocs, for completeness?

Reply:
Happy to take suggestions what to put there. I wanted to add some Rustdocs but couldn't come up with something that wasn't "well, launch like it says, duh." :)

Reply:
As someone who hasn't read the SBOM RFC recently, I would struggle to guess what belongs in a launch SBOM and what it's really for. As such, some rustdocs here seems necessary - and not obvious to me?

Reply:
I added changed the rustdocs for both via [1a9ac2c](https://github.com/heroku/libcnb.rs/pull/489/commits/1a9ac2cec66aeac47398a4b9e89f8c47f009111b). In the end I still believe the RustDoc here is just stating the obvious. The CNB spec itself does not describe the SBOM types beyond this in any meaningful way. 

Since I don't have any recommendations how to use these SBOM building blocks in a production buildpack, I'm hesitant adding any more documentation here that goes beyond the spec.